pkg/apis/autoscaling/v1/fleetautoscaler.go

```diff
@@ -195,6 +210,61 @@ type ListPolicy struct {
 	BufferSize intstr.IntOrString `json:"bufferSize"`
 }
 
+// Between defines the time period that the policy is eligible to be applied.
+type Between struct {
+	// Start is the datetime that the policy is eligible to be applied.
+	// This field must conform to RFC3339 format. If this field not set or is in the past, the policy is eligible to be applied
+	// as soon as the fleet autoscaler is running.
+	Start string `json:"start"`
```

Comment:
I'm going to assume that we can't make this a Go datetime object such that it will automatically get parsed appropriately?

We should be able to: https://pkg.go.dev/time#Time.MarshalJSON

Reply:
I had to use the k8's `time`, since the built-in Golang `time` doesn't have a `DeepCopy` functions.